test/scrabble_test.rb

```diff
@@ -1,13 +1,26 @@
 gem 'minitest'
-require_relative '../lib/scrabble'
+require './lib/scrabble'
 require 'minitest/autorun'
 require 'minitest/pride'
-require 'pry'
+require './lib/scrabble'
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
+
+
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
+  def test_instance_of_scrabble
+    game = Scrabble.new
+    refute_nil game
+  end
+  def test_whole_words
```

Comment:
This test name is not super descriptive & there are assertions that don't belong here; you're testing `''` & `""` which are not words and belong in their own tests.